x-pack/plugins/lens/public/indexpattern_datasource/fields_accordion.tsx

```diff
@@ -86,7 +88,22 @@ export const InnerFieldsAccordion = function InnerFieldsAccordion({
       id={id}
       buttonContent={
         <EuiText size="xs">
-          <strong>{label}</strong>
+          <p>
+            <strong>{label}</strong>
+            {!!helpTooltip && (
```

Comment:
Can we insert a blank here?

Reply:
Used some CSS rather than simple empty space to avoid space underlying:

<img width="422" alt="Screenshot 2020-12-14 at 15 47 30" src="https://user-images.githubusercontent.com/924948/102095424-d0a23180-3e23-11eb-9f90-b0b7b2d3ab10.png">

vs

<img width="417" alt="Screenshot 2020-12-14 at 15 30 09" src="https://user-images.githubusercontent.com/924948/102095444-d566e580-3e23-11eb-8164-83eecaefef7e.png">

Reply:
Nice!